parachain-template/node/Cargo.toml

```diff
@@ -21,6 +21,7 @@ path = "src/main.rs"
 
 [features]
 runtime-benchmarks = ["parachain-template-runtime/runtime-benchmarks"]
+try-runtime = []
```

Comment:
Shouldn't this include the `try-runtime` feature of the template runtime?

Reply:
correct, that would prevent us from using `pre-migrate` etc. great catch.

Reply:
Seems like you didn't adjust it, yet? :smile:

Reply:
it finally is now :D